internal/operands/metrics/resources.go

```diff
@@ -18,6 +21,14 @@ const (
 	PrometheusClusterRoleName    = "prometheus-k8s-ssp"
 	PrometheusServiceAccountName = "prometheus-k8s"
 	MetricsPortName              = "http-metrics"
+
+	TemplateValidatorMetricsServiceName = "template-validator-metrics"
```

Comment:
Please use this constant also in `template-validator/resources.go` file, so that we are sure that it is the same string.

Reply:
The above comment still applies.

The string `template-validator-metrics` is used to connect 2 resources in the cluster: `Service` and `ServiceMonitor`. It is a good code style if the 2 strings are also the same constant in the code. Then we can be sure, that there is not a typo in one of them.

Can you extract the constant to a different package, so it does not cause import loop?